app/controllers/reviews_controller.rb

```diff
@@ -10,10 +10,19 @@ def create
     redirect_to "/shelters/#{shelter.id}"
   end
 
+  def edit
+    @review = Review.find(params[:review_id])
+  end
+
+  def update
```

Comment:
Do you need instance variables here?  If this is just for the update thing I don't think you do....